clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/security/KubernetesNamedAccountCredentials.java

```diff
@@ -63,6 +63,14 @@ public KubernetesNamedAccountCredentials(AccountCredentialsRepository accountCre
     // TODO(lwander): what is this?
     this.requiredGroupMembership = requiredGroupMembership == null ? Collections.emptyList() : Collections.unmodifiableList(requiredGroupMembership);
     this.dockerRegistries = dockerRegistries != null ? dockerRegistries : new ArrayList<>();
+
+    for (int i = 0; i < this.dockerRegistries.size(); i++) {
+      LinkedDockerRegistryConfiguration registry = this.dockerRegistries.get(i);
```

Comment:
@lwander Did my comment make sense here? I was suggesting to use a java for-each loop instead of the counting loop (https://docs.oracle.com/javase/1.5.0/docs/guide/language/foreach.html).